rewrite-java/src/main/java/org/openrewrite/java/cleanup/FinalizePrivateFields.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.openrewrite.Cursor;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.Tree;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import java.util.Map;
+import org.openrewrite.java.tree.JavaType;
+import org.openrewrite.java.tree.Space;
+import org.openrewrite.marker.Markers;
+
+
+public class FinalizePrivateFields extends Recipe {
```

Comment:
Go ahead and add this to our [cleanup](https://github.com/openrewrite/rewrite/blob/0b994292372fce1f911ea7369a4aaf9afa2cb275/rewrite-java/src/main/resources/META-INF/rewrite/cleanup.yml#L18) and [common static analysis recipes](https://github.com/openrewrite/rewrite/blob/d408aaadddac717b00cc1f39e077bec84da51bc2/rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml#L18). Per Knut's helpful suggestion in the original issue, be sure it comes before the entry for RenamePrivateFieldsToCamelCase